examples/agents/slack-agent.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Slack Agent
+---
+
+Create a file `slack_agent.py` with the following code:
+
+```python slack_agent.py
+from phi.agent import Agent
+from phi.tools.slack import SlackTools
+
+
+slack_tools = SlackTools()
+
+agent = Agent(tools=[slack_tools], show_tool_calls=True)
+
+# Example 1: Send a message to a Slack channel
+agent.print_response("Send a message 'Hello from Phi!' to the channel #general", markdown=True)
```

Comment:
Lets make it more "agentic".
Something like "Send a warm welcome message to introduce yourself on the #general channel". 
Also give it some system instructions to define what it is